### PR TITLE
åsasieurin - project2 - newssite

### PR DESCRIPTION
**Pull request by sieurin**
First weeks project done and I am pretty happy with how it turned out. Of course I could have evolved it even more but I also try to learn to be fine with "good enough".

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,317 @@
...
+}
+
+.beige {
+  background-color: #dcd1b4;
+  color: #672f2a;
+}
+
+.beige p {
+  color: #672f2a;
+}
```
Comment (savannah-hayes) [inline code/style.css:216]: The class of the colors (.yellow, .blue, .plum...) for example is a little hard to understand what it does. Does it change the color of the article? the card maybe?

File: code/style.css [hunk 2]
```diff
@@ -1,4 +1,317 @@
...
+  animation-duration: 10s;
+  position: relative;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+}
+
+@keyframes scroll {
+  from { left: 100%; }
+  to { left: -80%; }
+}
```
Comment (savannah-hayes) [inline code/style.css:255]: Really nice animation effect! It is very smooth.

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,317 @@
...
+    flex-direction: row;
+  }
+
+  .hamburger-menu {
+    display: none;
+  }
+
+  .desktop-menu {
+    display: flex;
+  }
```
Comment (savannah-hayes) [inline code/style.css:296]: Very clear and descriptive css naming conventions here!

**Review by savannah-hayes — APPROVED** [review]
Great job on your news site Åsa, you seem to have met the black level requirements with your project and you went above and beyond. You should be proud of yourself!

Your website works as intended. There are no bugs. I can understand your code easily and I like your use of comments to help yourself and others who are reading your code. You made clear and descriptive comments. 

I have just made some small comments on your css and html files, merely suggestions. Great job!

File: code/index.html [hunk 4]
```diff
@@ -2,21 +2,191 @@
...
+
+        <div class="article-section">
+          <div class="article yellow">
+            <h4>The ideal female archetype</h4>
+            <a href="link to article">
+              <div class="article-image-wrapper">
+                <img
+                  src="./images/sergey-vinogradov-wXrXdt2_YGs-unsplash.jpg"
+                  class="article-image"
+                  alt="profile image"
```
[the remark this answers is not in the document]
Comment (sieurin) [reply code/index.html:78]: Totally agree :)

File: code/index.html [hunk 5]
```diff
@@ -2,21 +2,191 @@
...
+              <div class="article-image-wrapper">
+                <img
+                  src="./images/sergey-vinogradov-Jt6Mof2cLR0-unsplash.jpg"
+                  class="article-image"
+                  alt="man on a rock"
+                />
+              </div>
+              <p>BY ÅSA SIEURIN</p>
+            </a>
+          </div>
```
[the remark this answers is not in the document]
Comment (sieurin) [reply code/index.html:111]: That's absolutely a way to go, great input!

File: code/index.html [hunk 6]
```diff
@@ -2,21 +2,191 @@
...
+        </div>
+      </section>
+
+      <!-- End of the second article section -->
+
+      <h2 class="stay-in-touch">STAY IN TOUCH</h2>
```
[the remark this answers is not in the document]
Comment (sieurin) [reply code/index.html:181]: Ah, I see your point. Would you use a p?

**Comment by sieurin** [comment]
A huge thank you for all your kind comments and constructive feedback Savannah!

**Pull request closed without merging**